banking_system/maintenance.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <vector>
+
+#include "person.h"
+#include "i_account.h"
+#include "i_manager.h"
+
+namespace bank {
+    class Maintenance {
+    private:
+        std::vector<std::shared_ptr<Person>> m_clients;
+        std::vector<std::shared_ptr<IAccount>> m_accounts;
+    public:
+        Maintenance();
+
+        void listClients() const;
+        static void listClientAccount(std::vector<std::shared_ptr<IAccount>> const& accounts) ;
+        std::vector<std::shared_ptr<IAccount>> findClientAccounts(unsigned int clientId);
```

Comment:
Here and above, const modifications for base type have no effects in headers and can be placed only in .cpp files